src/components/AppForm.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import '../AppForm.css'
+
+class AppForm extends React.Component {
+    render() {
+        return (
+            <React.Fragment>
+            <form>
+                <input type="text" id="Playlist-name" placeholder="Name of the playlist"></input>
+                <button id="Edit">Edit</button>
+                <br></br>
+                <input type="text" id="Musician-name" placeholder="Musician's name"></input>
+            <br></br>
+            <button id="Add">Add</button>
+            <button id="Save">Save</button>
+            <button id="Delete">Delete</button>
+            </form>
+            <form id="musician">
+          
+        </form>
+        </React.Fragment>
+        )
+    }
```

Comment:
1. Please move this code into PlayListForm.js. It was created for that purpose. 
It ok to name a view component with a .js extension
2. Please replace React.Fragment by a simple div, It should work fine